### PR TITLE
feat: [wip] working on the frame of the window

### PR DESCRIPTION
**Pull request by tim02-k**
[Work in Progress] Start to working on the frame of the game window.

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -112,22 +113,20 @@ def remove_full_rows(self):
...
-        if self.tetrises >= self.level * 10 + 10:
+        if self.tetrises >= self.score.level * 10 + 10:
             self.tetrises = 0
-            self.level += 1
+            self.score.level += 1
             self.update_game_speed()
-            print(f"Reached Level: {self.level}")
-
-        print(f"Score: {self.score}")
+            print(f"Reached Level: {self.score.level}")
```
Comment (rpfaeffle) [inline src/main.py:129]: We should be good to remove this now.

File: src/core/application.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class Application(object):
     def __init__(self, setup=None, screen_size=None, title=None):
         # Set default variables
         if screen_size is None:
-            screen_size = [510, 510]
+            screen_size = [520, 520]
```
Comment (rpfaeffle) [inline src/core/application.py:16]: This wouldn't be necessary, but we can keep it.

File: src/main.py [hunk 3]
```diff
@@ -15,10 +16,9 @@ class Tetris(Render):
     def __init__(self, cx):
         self.blocks: list[list[Optional[Block]]] = []
         self.lines: list[Line] = []
+        self.score = Score(1, 0)
```
Comment (rpfaeffle) [inline src/main.py:19]: Maybe we can add default values on initialisation

File: src/components/score.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+from components.block import Block
+
+
+class Score(Component):
+
+    def __init__(self, level: int, score: int):
+        super().__init__()
+        self.level = level
+        self.score = score
+        self.background = Block((0, 0, 0), 0, 0, 240, 100)
```
Comment (rpfaeffle) [inline src/components/score.py:13]: This is not used, so we can remove it

**Review by rpfaeffle — CHANGES_REQUESTED** [review]
Good work. I made some comments about things that could be changed before merging.

File: src/components/score.py (hunk 4 above)
Comment (tim02-k) [reply src/components/score.py:13]: We need this part, i tested the program when it's removed and it doesn't worked

File: src/core/application.py (hunk 2 above)
Comment (tim02-k) [reply src/core/application.py]: Ok, i  keep this at 520

**Pull request merged**